[2022-09-21T22:23:33.822+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag_example.create_airflow_demo_schema manual__2022-09-21T22:23:31.726194+00:00 [queued]>
[2022-09-21T22:23:33.831+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag_example.create_airflow_demo_schema manual__2022-09-21T22:23:31.726194+00:00 [queued]>
[2022-09-21T22:23:33.832+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-21T22:23:33.834+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2022-09-21T22:23:33.836+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-21T22:23:33.845+0000] {taskinstance.py:1389} INFO - Executing <Task(PostgresOperator): create_airflow_demo_schema> on 2022-09-21 22:23:31.726194+00:00
[2022-09-21T22:23:33.853+0000] {standard_task_runner.py:52} INFO - Started process 63167 to run task
[2022-09-21T22:23:33.857+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag_example', 'create_***_demo_schema', 'manual__2022-09-21T22:23:31.726194+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_demo.py', '--cfg-path', '/tmp/tmpu2__vxw8', '--error-file', '/tmp/tmp3jgb0hr3']
[2022-09-21T22:23:33.859+0000] {standard_task_runner.py:80} INFO - Job 16: Subtask create_***_demo_schema
[2022-09-21T22:23:33.925+0000] {task_command.py:371} INFO - Running <TaskInstance: postgres_operator_dag_example.create_airflow_demo_schema manual__2022-09-21T22:23:31.726194+00:00 [running]> on host 49fa11dd9034
[2022-09-21T22:23:34.001+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=jsanchez
AIRFLOW_CTX_DAG_ID=postgres_operator_dag_example
AIRFLOW_CTX_TASK_ID=create_***_demo_schema
AIRFLOW_CTX_EXECUTION_DATE=2022-09-21T22:23:31.726194+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-21T22:23:31.726194+00:00
[2022-09-21T22:23:34.014+0000] {base.py:68} INFO - Using connection ID 'qa_ai_external_db' for task execution.
[2022-09-21T22:23:34.928+0000] {sql.py:315} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS ***_demo AUTHORIZATION jsanchez, parameters: None
[2022-09-21T22:23:35.233+0000] {postgres.py:94} INFO - NOTICE:  schema "***_demo" already exists, skipping

[2022-09-21T22:23:35.263+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag_example, task_id=create_***_demo_schema, execution_date=20220921T222331, start_date=20220921T222333, end_date=20220921T222335
[2022-09-21T22:23:35.336+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-21T22:23:35.376+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
